plugconf/plugconf.go

```diff
@@ -116,8 +116,6 @@ func (pi *ParsedInfo) GeneratePlugconf() ([]byte, error) {
 		buf.WriteString(f)
 	}
 
-	buf.WriteString("\n")
```

Comment:
Please rebase your branch on `devel`.
This PR wrongly reverts #276 change.

Reply:
Sorry for that mistake.

I have rebased and pushed again.